src/openrct2/interface/InteractiveConsole.cpp

```diff
@@ -744,9 +739,24 @@ static int32_t ConsoleCommandGet(InteractiveConsole& console, const arguments_t&
             console.WriteLineWarning("Invalid variable.");
         }
     }
-    return 0;
 }
-static int32_t ConsoleCommandSet(InteractiveConsole& console, const arguments_t& argv)
+
+template<typename TAction, typename... TArgs>
+static void ConsoleSetVariableAction(InteractiveConsole& console, std::string var, TArgs&&... args)
+{
+    auto action = TAction(std::forward<TArgs>(args)...);
+    action.SetCallback([&console, var](const GameAction*, const GameActions::Result* res) {
+        if (res->Error != GameActions::Status::Ok)
+            console.WriteLineError(String::stdFormat("set %s command failed, likely due to permissions.", var.c_str()));
```

Comment:
`stdFormat` should be `StdFormat`. Even this branch doesn't have `stdFormat` so I actually have no idea how this is passing CI.

Reply:
I don’t know how this got messed up but CI did not accept this PR without the change to ‘stdFormat’

Reply:
Here is the relevant run: https://github.com/OpenRCT2/OpenRCT2/actions/runs/12340982562

It's odd because the only reference to `stdFormat` in the entire git history is your commit.

Reply:
Oh, its because CI is adding this branch on top of the latest `develop` which has https://github.com/OpenRCT2/OpenRCT2/pull/23384

If you just look at the history in this branch it doesn't yet have it.